2019-06-28-Nitrogen-grid-optimization/send_mols_to_server.py

```diff
@@ -71,7 +77,7 @@ def send_qm_job(ptl_mol, nitrogen, nitrogen_i,  mol):
                     "scans": [{
                         "type": "dihedral",
                         "indices": list(indices),
-                        "steps": [-40, -36, -32, -28, -24, -20, -16, -12, -8, -4, 0, 4, 8, 12, 16, 20, 24, 28, 32, 36, 40],
+                        "steps": [-50, -48, -46, -44, -42,-40, -36, -32, -28, -24, -20, -16, -12, -8, -4, 0, 4, 8, 12, 16, 20, 24, 28, 32, 36, 40, 42, 44, 46, 48, 50],
```

Comment:
Is there a reason for a finer grid at the edges here? As a note we do not have technology which can stop a gridoptimizaiton if the energy becomes too large.

@yudongqiu In torsiondrive for 2d-scans how do you stop the propagation if the energy is too high? In the 1-D case it is easy to see the cap. For 2-D we have something like where `x` is complete and `z` is too high. We can view a propagation like so:

```
o o o o o
o o x z o
o o o o o 
```

```
o o x o o
o x x z o
o o x o o 
```

```
o x x x o
x x x z o
o x x x o 
```

We can quickly see that in the 2-D case we can overcome bad area even though we might not want to?